Trimble.Modus.Components/Controls/TMCheckBox/TMCheckBox.cs

```diff
@@ -0,0 +1,199 @@
+﻿using Microsoft.Maui.Controls;
+using System.Drawing;
+using Trimble.Modus.Components.Enums;
+
+namespace Trimble.Modus.Components
+{
+    public class TMCheckBox : ContentView
+    {
+        #region Fields
+        private Label _label;
+        private Image _checkbox;
+        private int _default_width = 24, _default_height = 24, _large_width = 32,  _large_height = 32;
```

Comment:
To follow naming convention. Private members should be in `_camelCasing` with a leading underscore. Underscores inbetween should be removed

Reply:
Renamed